libs/sdk-backend-spi/src/organization/notificationChannels/index.ts

```diff
@@ -0,0 +1,50 @@
+// (C) 2023-2024 GoodData Corporation
+
+import { IWebhookDefinition, ICreateWebhookDefinition } from "@gooddata/sdk-model";
+
+/**
+ * This service provides access to organization notifications channels.
+ *
+ * @alpha
+ */
+export interface IOrganizationNotificationChannelService {
+    /**
+     * Get all webhooks
+     *
+     * @returns Promise resolved with array of webhooks.
+     * @throws In case of error.
+     */
+    getWebhooks(): Promise<IWebhookDefinition[]>;
+
+    /**
+     * Get webhook by id
+     *
+     * @param id - id of the webhook
+     * @returns Promise resolved with webhook definition or undefined if not found.
+     */
+    getWebhook(id: string): Promise<IWebhookDefinition | undefined>;
```

Comment:
Is `undefined` proper return type? I think in case of "not found", other methods currently throw 404 error.